pkgs/development/python-modules/umi_tools/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gcc
+, pythonOlder
+, pythonPackages
+, zlib
+}:
+
+buildPythonPackage rec {
+  pname = "umi_tools";
+  version = "1.1.2";
+
+  format = "setuptools";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    sha256 = "07yl14jiziksiplnqyfl08axpg8wrlk13m8szqkx4zpw6070yih4";
+  };
+
+  nativeBuildInputs = [ gcc zlib ];
+  propagatedBuildInputs = with pythonPackages; [
```

Please add these inputs to the lambda parameters at the top of the file. Otherwise they are not possible to be overridden by downstream users.

```suggestion
  propagatedBuildInputs = [
```